.gitattributes

```diff
@@ -1 +1,6 @@
 *.bat	text eol=crlf
+.circleci/config.yml linguist-generated=true
+.github/workflows/periodic-pytorch-win-vs2019-cuda11-cudnn8-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cpu-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cuda10-cudnn7-py3.yml linguist-generated=true
+.github/workflows/pytorch-win-vs2019-cuda11-cudnn8-py3.yml linguist-generated=true
```

Comment:
is there no way to just have a single pattern that matches every `.github/workflows/pytorch-*.yml` file?